src/App.vue

```diff
@@ -86,6 +86,11 @@
       :dialog="getAmalgamateNowErrorStatus"
       @close="closeAmalgamateNowErrorDialog()"
     />
+    <ContinuationInErrorDialog
```

Comment:
New error dialog to show when continuation in action fails.

Reply:
With 3 similar dialogs now, would it be easy(ish) to combine them?